_pages/index.md

```diff
@@ -16,48 +16,104 @@ Login.gov is a standard SAML identity provider, adhering to the [Web Browser SSO
 
-## Configuration
+## Integration
 
-Our sandbox environment is available for testing your service provider application. View our [metadata endpoint](https://idp.dev.login.gov/api/saml/metadata) for all the pertinent information.
+Integration is a two-step process; the Service Provider and Identity Provider (login.gov) must both be configured to integrate with one another. 
 
-All authentication and logout requests must be signed. We require RSA SHA-256 signatures embedded with the authentication and logout requests. Here's some of the key information you'll need:
+### 1. Configuring your Service Provider
+
+Our sandbox environment is available for testing integration with your service provider application. View our [metadata endpoint](https://idp.dev.login.gov/api/saml/metadata) for all of the pertinent information.
+
+To view an example of how a Service Provider would be configured, check out our [Demo SP Application's configuration](https://github.com/18F/identity-sp-rails/blob/master/config/initializers/omniauth.rb).
+
+All authentication and logout requests must be signed. We require RSA SHA-256 signatures embedded with the authentication and logout requests. Here's some of the key information you'll need to configure on your side:
 
 ```xml
-<!-- NameID Format: v4 UUID https://tools.ietf.org/html/rfc4122 -->
 <NameIDFormat>urn:oasis:names:tc:SAML:2.0:nameid-format:persistent</NameIDFormat>
+```
+**NameID Format** - The NameID is the unique identifier used to identify a user across multiple sessions. The format is the standard v4 random UUID (Universally Unique IDentifier) in compliance with [RFC 4122](https://tools.ietf.org/html/rfc4122).
+<br>
 
+```xml
 <SingleSignOnService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect" Location="https://idp.dev.login.gov/api/saml/auth" />
+```
+**Login service URL and Binding** - This is the endpoint where authentication requests are sent to login.gov. (aka Single Sign-on Service)
+<br>
 
+```xml
 <SingleLogoutService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST" Location="https://idp.dev.login.gov/api/saml/logout" />
-
-<KeyInfo xmlns="http://www.w3.org/2000/09/xmldsig#">
-  <ds:X509Data>
-    <ds:X509Certificate>
-      <!-- Get the latest certificate from the Metadata endpoint -->
-    </ds:X509Certificate>
-  </ds:X509Data>
-</KeyInfo>
 ```
+**Logout service URL and Binding** - The single logout service URL is used by to contact the Single logout profile. (aka Single Logout Service)
+<br>
+
+**x509 Public Certificate** - The public certificate is used to validate the authenticity of SAML requests received from login.gov. Minimum 2048 bits.
+<br>
+
+### 2. Configuring login.gov to accept your Service Provider
+
+In order to successfully integrate, your Service Provider application details must be whitelisted within login.gov. We require the following details about your service provider application:
```

Comment:
agreed. I was thinking of adding an official email address, but I think it's just easier to assume our partners know how to get in touch with us if they are ready to test integration. 

In the glorious future, this will be "log in to the Dashboard and configure these parameters"